libc/src/__support/OSUtil/gpu/quick_exit.cpp

```diff
@@ -6,17 +6,14 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef LLVM_LIBC_SRC___SUPPORT_OSUTIL_GPU_QUICK_EXIT_H
-#define LLVM_LIBC_SRC___SUPPORT_OSUTIL_GPU_QUICK_EXIT_H
```

Comment:
Wonder how these got here, probably some victim of copy paste.

Reply:
it happens